package.json

```diff
@@ -15,6 +15,7 @@
   "author": "visualfanatic",
   "main": "index.js",
   "dependencies": {
-    "svgo": "^0.7.1"
+    "svgo": "^0.7.1",
+    "vue-template-compiler": "^2.4.2"
```

Comment:
Should this be a peer dependency or something? It didn't match the version of Vue I had installed on my project.